test/state-alea.test.ts

```diff
@@ -0,0 +1,69 @@
+import { RngStateAlea, masher } from "../src/state-alea";
+import "./matchers";
+
+describe("can generate random numbers", () => {
+    it("generates known case", () => {
+        // just generate two numbers and show they're different and
+        // that they don't change; this should always just hold.
+        const s = new RngStateAlea(42);
+        const u1 = s.random();
+        const u2 = s.random();
+        expect(u1).toBe(0.6848634963389486);
```

Comment:
Could also generate these numbers from one of the reference algorithms? But the code does all look the same to my eyes :+1:

Reply:
good point, will sort something out there

Reply:
ok, this now checks against the node-alea implementation